cookbooks/bcpc-hadoop/recipes/hbase_config.rb

```diff
@@ -168,6 +168,12 @@
   "-XX:+PrintGCApplicationStoppedTime -XX:+UseCompressedOops " + 
   "-XX:+PrintClassHistogram -XX:+PrintGCApplicationConcurrentTime"
 
+if node[:bcpc][:hadoop][:kerberos][:enable] == true then
+ env_sh[:HBASE_OPTS] = '"$HBASE_OPTS" -Djava.security.auth.login.config=/etc/hbase/conf/hbase-client.jaas'
```

Comment:
Should this be:
`env_sh[:HBASE_OPTS] = '"$HBASE_OPTS -Djava.security.auth.login.config=/etc/hbase/conf/hbase-client.jaas"'?